docs/sources/get-started/architecture.md

```diff
@@ -9,54 +9,97 @@ aliases:
 ---
 # Loki architecture
 
-## Multi-tenancy
+Grafana Loki has a microservices-based architecture and is designed to run as a horizontally scalable, distributed system.
+The system has multiple components that can run separately and in parallel.
+Grafana Loki's design compiles the code for all components into a single binary or Docker image.
+The `-target` command-line flag controls which component(s) that binary will behave as.
 
-All data, both in memory and in long-term storage, may be partitioned by a
-tenant ID, pulled from the `X-Scope-OrgID` HTTP header in the request when Grafana Loki
-is running in multi-tenant mode. When Loki is **not** in multi-tenant mode, the
-header is ignored and the tenant ID is set to "fake", which will appear in the
-index and in stored chunks.
+To get started easily, run Grafana Loki in "single binary" mode with all components running simultaneously in one process, or in "simple scalable deployment" mode, which groups components into read, write, and backend parts.
+
+Grafana Loki is designed to easily redeploy a cluster under a different mode as your needs change, with no configuration changes or minimal configuration changes.
+
+For more information, refer to [Deployment modes]({{< relref "./deployment-modes" >}}) and [Components]({{< relref "./components" >}}).
+
+![Loki's components](../loki_architecture_components.svg "Loki's components")
+
+## Storage
+
+Loki stores all data in a single object storage backend, such as Amazon Simple Storage Service (S3), Google Cloud Storage (GCS), Azure Blob Storage, among others.
```

Comment:
`all data (both chunks and index)` should we be more explicit?

Reply:
It's basically mentioned in the next line:
> This mode uses an adapter called **index shipper** (or short **shipper**) to store index (TSDB or BoltDB) files the same way we store chunk files in object storage.